src/tasks/containers/TasksPage.test.tsx

```diff
@@ -42,6 +42,10 @@ const localHistory = createMemoryHistory({initialEntries: ['/']})
 
 withRouterProps.match.params.orgID = orgs[0].id
 
+jest.mock('src/shared/constants/index', () => ({
+  CLOUD: true,
+}))
+
```

Comment:
does this affect all the other tests in this file? Should we move this mock into your specific test that requires this variable to be set?

Reply:
Even better news, we dont even need it anymore after my latest changes!